dist/preview release/what's new.md

```diff
@@ -42,9 +42,8 @@
 - Added right click options to create PBR and Standard Materials ([Deltakosh](https://github.com/deltakosh))
 - Added support for recording GIF ([Deltakosh](https://github.com/deltakosh))
 - Popup Window available (To be used in Curve Editor) ([pixelspace](https://github.com/devpixelspace))
-- Curve Editor to view current entity's animations (Read only) ([pixelspace](https://github.com/devpixelspace))
+- Curve Editor to view selected entity's animations (Read only) ([pixelspace](https://github.com/devpixelspace))
```

Comment:
this should be in the Major feature list :) (upper in the file)

Reply:
ah.. ok. 😃